spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    before(:each) { get posts_path }
+
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it "renders 'index' template" do
+      expect(response).to render_template('index')
+    end
+
+    it 'the response body includes correct placeholder text' do
+      expect(response.body).to include('Find me in app/views/post/index.html.erb')
+    end
+  end
+
+  describe 'GET /show' do
+    before(:each) { get user_path(1) }
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+    it "renders 'show' template" do
+      expect(response).to render_template('show')
```

Comment:
- [ ] Hi @aron-helu, some of your test cases are failing. Let's kindly fix them.


![Screenshot from 2022-11-01 19-01-24](https://user-images.githubusercontent.com/63087491/199305718-7150cc14-6cc0-4ba0-9011-f1a1f60398e2.png)

Reply:
Already implemented it! Thanks for the feed back! :smile: